pkg/envconf/config_test.go

```diff
@@ -85,3 +87,21 @@ func TestConfig_New_WithIgnorePanicRecovery(t *testing.T) {
 		t.Error("expected ignore-panic-recovery mode to be enabled when -disable-graceful-teardown argument is passed")
 	}
 }
+
+func TestRandomName(t *testing.T) {
+	const maxLen = 64
+
+	t.Run("no prefix yields random name without dash", func(t *testing.T) {
+		out := RandomName("", int(rand.Int31n(maxLen)))
```

Comment:
Do we really need to use the `math.rand` based value for the input? Can it not be a pre-defined number?

Reply:
👍 Changed to use a constant prefix length